spec/spec_helper.rb

```diff
@@ -18,6 +18,6 @@
 #++
 
 # -- load spec_helper from OpenProject core
-require "spec_helper"
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.